LoginPage.jsx

```diff
@@ -0,0 +1,27 @@
+import React, { useState } from 'react';
+import { useHistory } from 'react-router-dom';
+import { loginUser } from './api';
+
+function LoginPage() {
+  const [email, setEmail] = useState('');
+  const [password, setPassword] = useState('');
+  const history = useHistory();
+
+  const handleSubmit = async (e) => {
+    e.preventDefault();
+    const user = await loginUser(email, password);
+    if (user) {
```

Comment:
The user object is not being used. If it's not needed, consider removing it to reduce complexity.